### DISCUSSION TITLE
vercel guides not opening

### DISCUSSION THREAD

**Discussion by raiadi2008** [vercel/community#3077] [Help] [answered]
### Summary

https://vercel.com/guides/nextjs-multi-tenant-application

while opening this URL or any URL with starting pathname /guides I am getting the following error

```Application error: a client-side exception has occurred (see the browser console for more information).```

### Steps to Reproduce

just go to url

Not sure if this is right place to post but i am not getting any other place to post it.

**Comment by leerob** [comment, marked as answer]
Thanks! We're working on a fix for this as we speak.
